COMETwebapp/Model/DataChart.cs

```diff
@@ -43,5 +45,20 @@ public class DataChart
         /// The domain of expertise of the data
         /// </summary>
         public string Domain { get; set; }
```

Comment:
what data of the Domain and other properties in this class is being used? i am a bit surprised to see strings, should you not use Guid?

Reply:
The object DataChart is used to build graphs : here Domain is used to sort data in the graphs and appears in the legend. If I put Guid, I can't choose what appears in the legend...

Reply:
This is also why I am not satisfied of my work. I don't know how to directly used Parameter objects in the graphs without building new DataChart objects.